### ISSUE TITLE
Causes lag online

### ISSUE THREAD

**Issue by CharlieD93** [ccMSC/ckb#608]
Hi, I downloaded the driver and it made my mouse and keyboard work, but when I tested them on some games (League of Legends and Guild Wars 2) I found that it would make my ping vary between 30 up to 200, and would jump up in step of 10-15ms. I then uninstalled and the problem stopped.

**Comment by joshliptzin** [comment]
I came here to mention this too. Here is a screenshot of pinging google.com with CKB open.

![screen shot 2017-08-25 at 6 36 07 pm](https://user-images.githubusercontent.com/1000114/29735104-70948ed0-89c4-11e7-8fd9-2396aaa6c184.png)

I had actually installed CKB a few weeks ago, hoping to use a corsair mouse that I liked better instead of my original mouse. I couldn't get it to work (forward/back buttons wouldn't work) and on top of that the mouse would randomly jump around the screen, so I went back to my old mouse. Forgot to uninstall CKB and only figured out it was the source of my frustrating ping spikes after several hours of debugging. I would not recommend anyone install this software at this time.

**Comment by sunshine7913** [comment]
Yeah me too. Im having lags that I can not move my mouse properly. When I quit CKB, it works normally.
